database/read-and-write.js

```diff
@@ -123,3 +123,18 @@ function toggleStar(postRef, uid) {
   });
 }
 // [END rtdb_social_star_transaction]
+
+/**
+ * @param {string} uid 
+ * @param {string} key 
+ */
+// [START rtdb_social_star_increment]
```

Comment:
Can you also add the equivalent `database-next` snippet? I just updated all of those to the new modular API this morning so it should be in good shape for you to tack one more on.

Reply:
I completely forgot about that one. Thanks for catching! 👍